components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -27,7 +27,7 @@
     </chef-page-header>
     <section class="page-body" [hidden]="showTab('rules')">
       <form [formGroup]="projectForm">
-        <chef-form-field id="update-name">
+        <chef-form-field id="update-name" [watch]="projectForm.controls.name.value" [control]="projectForm.controls.name">
```

Comment:
How it is used: by just indicating the thing to watch and its control...

Reply:
Is it necessary to separate them? Are there cases where `[watch]` is not `[control]+.value`?

Reply:
1. Yes. 2. No
But--and it is a big but--the change detection strategy needs not just a Thing, but the Angular-Thing, to watch. In the .ts file, that is specified by the `@Input`, not any of its descendents (like `.value`). So that's why `watch` is needed. And `control` is needed because, from the value, there is no way to discern its parent control.
I will try to add that to the infile documentation.